src/components/notifications/PageNotifications.vue

```diff
@@ -68,13 +68,15 @@ export default {
       if (this.dataLoaded === true) {
         // loads new portion
         this.dataLoaded = false
+        const lastTimestamp = this.notifications[this.notifications.length - 1]
+          .timestamp
+        const dateLastTimestamp = new Date(lastTimestamp)
         this.$apollo.queries.notifications.fetchMore({
           // New variables
           variables: {
-            addressObjects: this.session.allSessionAddresses,
             // get notifications that are older then the last one
-            timestamp: this.notifications[this.notifications.length - 1]
-              .timestamp,
+            timestamp: dateLastTimestamp.toISOString(),
```

Comment:
so the timestamp we send is not an ISOString?

Reply:
this feels like weird logic and should be addressed in the API @michielmulders

Reply:
Yes, I can open an issue for this. It is weird. The timestamps in the DB are this format: 
2020-06-05T21:26:09.548319+00:00. Which looks very similar to ISO but are not exactly ISO (2020-06-06T15:40:23.925Z)